README.md

```diff
@@ -128,12 +128,15 @@ $ omf install https://github.com/jhillyerd/plugin-git
 | glr          | `git pull --rebase`                                  |
 | gp           | `git push`                                           |
 | gp!          | `git push --force-with-lease`                        |
+| gpo          | `git push origin`                                    |
 | gpv          | `git push --no-verify`                               |
 | gpv!         | `git push --no-verify --force-with-lease`            |
 | ggp          | push origin _current-branch_                         |
+| ggf          | `ggp --force`                                        |
+| ggfl         | `ggp --force-with-lease`                             |
+| gpu          | `ggp --set-upsteam`                                  |
```

Comment:
`gpu` exists as a function already.  I'm not sure if the functionality is any different, as some of the functions are quite old and git built in options may have made them obsolete.

Reply:
What I did was to make `gpu` an abbreviation instead (removed the function) since its definition is pretty much identical to `ggp` - which doesn't really need a dedicated function to serve that. Aliasing it to `ggp --set-upstream` should work for most cases (I'm well aware of the argument shifting, but still haven't been able to think up of a case where it fails). The only difference is `--set-upstream` is now pushed to the shifted - after the branch name.

```
git push --set-upstream origin (current_branch)
# versus
git push origin (current_branch) --set-upstream
```